stl/inc/chrono

```diff
@@ -619,8 +618,61 @@ namespace chrono {
             // but the initial value of _Ctr could be large.
             const long long _Whole = (_Ctr / _Freq) * period::den;
             const long long _Part  = (_Ctr % _Freq) * period::den / _Freq;
-            return time_point(duration(_Whole + _Part));
+            return _Whole + _Part;
+        }
+
+        _NODISCARD static time_point now() noexcept { // get current time
+#if (defined(_M_IX86) || defined(_M_X64) || defined(_M_ARM) || defined(_M_ARM64)) && !defined(_M_CEE_PURE)
+            // Implement atomics avoiding <atomic> header dependency
+            static struct {
+                volatile long long _Freq        = LLONG_MAX;
+                volatile long long _Ctr_base    = LLONG_MAX;
+                volatile long long _Result_base = LLONG_MAX;
+            } _Cached;

```

Comment:
I assume that this avoids triggering magic statics because the initializers are compile-time constants?

Reply:
I double checked. Unfortunately, this fails to avoid magic static with `/Od`, whereas avoidance works with `/O2`. Breaking the structure makes avoidance of magic static stable https://godbolt.org/z/4cSNtj

Need to fix this.

(I made conclusion that grouping into struct does not break magic static, because I was testing only release build.
Grouping into struct helps when magic static is actually used for initialization instead of atomic exchange - thee separate magic statics perform worse, whereas one is on the same level with atomic exchange on x64 and is better on x86)

Reply:
Did someone report the constant initialization bug?

Reply:
Aha, apparently there's nothing to report. Except report to https://godbolt.org/ to update compiler. 19.25.28614, which is current non-preview, already fixed that.

Reply:
I still prefer keeping non-struct version to stand away from this.